lib/stimulus_reflex/reflex.rb

```diff
@@ -12,21 +12,21 @@ class StimulusReflex::Reflex
   include CableReady::Identifiable
 
   attr_accessor :payload, :headers
-  attr_reader :channel, :reflex_data, :broadcaster
+  attr_reader :channel, :data, :broadcaster
```

Comment:
Thanks for taking this on @brunoenten!

Can we instead rename the keyword argument from `data:`  to `reflex_data:` here instead of renaming it throughout the `Reflex` class:

https://github.com/stimulusreflex/stimulus_reflex/blob/3137a0f36a27dc26ee271eee5c17c3a96dce280d/lib/stimulus_reflex/reflex_factory.rb#L13-L15

This way the internal API doesn't change we also don't have to update the currently failing tests.

Thank you!

Reply:
Done! I added an alias for data, with the same goal of impacting as little existing code as possible.